cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/LoginPage.java

```diff
@@ -76,12 +96,11 @@ public LoginPage(WebDriver driver) {
         // Initialize form and add elements
         formLogin = new Form(driver);
 
-        formLogin.setInput(IN_USERNAME_ID, IN_USERNAME_NAME);
```

Comment:
Medium impact: other PRs have shown the LoginPage.login method used, which directly makes use of the formLogin.
Suggestion: Bring back the formLogin.

Reply:
formLogin is back after accidental removal and all login tests passing